### ISSUE TITLE
Wagnerwagner\Merx\ProductList::__set(): Return type must be void when declared

### ISSUE THREAD

**Issue by raphdurrer** [wagnerwagner/merx#36]
Hey after PHP Update i get this error.
`Wagnerwagner\Merx\ProductList::__set(): Return type must be void when declared`

`PHP Version: 7.4.16`

I have this error on every merx plugin installation.

I think there is some work todo for making merx plugin future proof.
Thanks for Fixing this as soon as possible.

Cheers Raphael

**Comment by raphdurrer** [comment]
OK... i just found out it is just for this special PHP Version. After changing to PHP8 it seems to work fine again.

**Comment by hroma** [comment]
Hi, 

I also get "Failed to load resource: the server responded with a status of 500 (Internal Server Error)" with php 8.0.3.

<b>Fatal error</b>:  Wagnerwagner\Merx\ProductList::__set(): Return type must be void when declared in <b>C:\test\site\plugins\merx\src\productList.php</b> on line <b>82</b><br />
{"status":"error","exception":"Whoops\\Exception\\ErrorException","code":64,"message":"Wagnerwagner\\Merx\\ProductList::__set(): Return type must be void when declared","details":null,"file":"\\test\\site\\plugins\\merx\\src\\productList.php","line":82}

**Comment by hroma** [comment]
Also tested with PHP Version 7.3.10. With this version I get:

Argument 2 passed to Kirby\Cms\App::trigger() must be of the type array, object given, called in Z:\web_dev\base\bs.com\site\plugins\merx\src\cart.php on line 22

**Comment by jaro-io** [comment]
hey lovely @wagnerwagner team,
thank you so much for your hard work on this beautiful plugin! 🤍 ✨

it's strange but when trying to add products to the cart, i am seeing the same error from above with php `8.0.14` on kirby `3.6.1.1`. can you please check that?

thanks!
☺️ 

```
Whoops\Exception\ErrorException thrown with message "Wagnerwagner\Merx\ProductList::__set(): Return type must be void when declared"

Stacktrace:
#2 Whoops\Exception\ErrorException in /site/plugins/merx/src/ProductList.php:83
#1 Whoops\Run:handleError in /kirby/vendor/filp/whoops/src/Whoops/Run.php:486
#0 Whoops\Run:handleShutdown in [internal]:0
```

**Comment by tobiasfabian** [comment]
This issue is fixed in [1.6.0.beta-1](https://github.com/wagnerwagner/merx/releases/tag/1.6.0-beta.1)
